bsdlib/doc/gnss_extension.rst

```diff
@@ -147,11 +147,57 @@ Satellites with elevation angle less than the threshold are automatically exclud
+   | Allowed values:
+
+   | :c:type:`NRF_GNSS_USE_CASE_SINGLE_COLD_START`
+   | :c:type:`NRF_GNSS_USE_CASE_MULTIPLE_HOT_START`
+   | :c:type:`NRF_GNSS_USE_CASE_NORMAL_ACCURACY`
+   | :c:type:`NRF_GNSS_USE_CASE_LOW_ACCURACY`
+
+This bitmask is used to specify the combination of the target GNSS performance (single cold or multiple hot start)
+and accuracy (normal or low).
+
+..Note::
+  Normal and Low accuracy is available in MFW 1.2.2 and later
+
+Low Accuracy:
+
+Enabling low accuracy mode will result in the GNSS receiver having a looser acceptance criterion
+for a fix (i.e. error in calculation vs actual position can be larger than Normal Accuracy mode).
+In addition, in the Low Accuracy mode the GNSS receiver may use only 3 satellites to determine a
+fix (in Normal Accuracy mode, it needs 4 or more satellitesi).
+
+For a possible 3 satellite fix to take place, one of the following criteria must be in place first
+and will be valid for a time window of 24 hour:
+
+ | 1) At least 1 fix must have been taken place using 5 or more satellites. You initiate the GPS to start Low accuracy mode, but the radio will not be able to utilize 3 satellite fixing before the criteria above has taken place. In the following time window, any fix that is done using 5 or more satellites will reset the time window and extend the 24-hour period.
+ |
+ | 2) Injecting Altitude (in meter), Altitude uncertainty (0) , and Confidence (100 percent) to the GPS radio using agps data location struct as described below. To be able to stay in Low Accuracy mode this step has to be repeated at least every 24 hour.
+ |
+ | **Caution:** The Altitude in meter has to have an accuracy in the range of +/- 10’s of meter of your actual height. An erroneous altitude will result in a quite sever error in 3 satellite position fix calculation.
+
+.. code-block:: c
 
-The use case parameter determines the target performance.
-A value of 0 indicates single cold start and one indicates multiple hot starts.
+    typedef struct
+    {
+        int32_t latitude          = 0;     // Not known, set to 0
+        int32_t longitude         = 0;     // Not known, set to 0
+        int16_t altitude          = meter; // Your actual height in meter
+        uint8_t unc_semimajor     = 127;   // Both set to 127 when latitude
+        uint8_t unc_semiminor     = 127;   // and longitude are unknown
+        uint8_t orientation_major = 0;     // Not known, set to 0
+        uint8_t unc_altitude      = 0;     // Set to 0
+        uint8_t confidence        = 100;   // Set to 100
+    } nrf_gnss_agps_data_location_t;
+
+If the actual altitude of where the device is located changes vs the altitude stored in the
+GNSS engine (e.g. the device is moving around), the accuracy of the 3 sat fix will be degraded.
+
+This Low Accuracy mode is not the same as the 2D fix you see documented in the NMEA reports.
+All fixes, including the Low Accuracy fix, will be reported as 3D fix.
```

Comment:
`All fixes, including the Low Accuracy fix, will be reported as 3D fix.` -> `All fixes, including Low Accuracy fixes, will be reported as 3D fixes.`